integration/dc_fixture_test.go

```diff
@@ -44,6 +50,22 @@ func (f *dcFixture) dockerContainerID(name string) (string, error) {
 	return cID, nil
 }
 
+func (f *dcFixture) dockerCreatedAt(name string) string {
+	out := &bytes.Buffer{}
+	cmd := f.dockerCmd([]string{
+		"ps", "-q", "-f", fmt.Sprintf("name=%s", name), "--format", "{{.CreatedAt}}",
+	}, out)
+	err := cmd.Run()
+	if err != nil {
+		f.t.Fatal(fmt.Errorf("dockerCreatedAt: %v", err))
+	}
+	cID := strings.TrimSpace(out.String())
```

Comment:
these variable names and error message 🤔

Reply:
these...are pretty idiomatically standard Go; can you elaborate more on why you don't like them?